components/Page.js

```diff
@@ -3,9 +3,9 @@ import Footer from './Footer';
 
 export default function Page({ children }) {
 	return (
-		<div>
+		<div className='Page-component'>
```

Comment:
Close. BEM asks for two separating characters. '--'

Reply:
fixed